### PR TITLE
Revisions for Sprint-11

### PR DESCRIPTION
**Pull request by middle8media**
changes the medium screen size overflow on `mode-context.init` from `auto` to `visible`
fixes https://github.com/ushahidi/platform-pattern-library/issues/78

---
This change is [Reviewable](https://reviewable.io/reviews/ushahidi/platform-pattern-library/82)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ushahidi-platform-pattern-library",
   "description": "Ushahidi Platform Pattern Library",
-  "version": "3.5.0",
+  "version": "3.5.1",
```
Comment (rjmackay) [inline package.json]: @middle8media This should have been `3.5.1-pre.1` or something similar. It shouldn't get a real version until we're absolutely ready to go to production. Once a version is pushed to npm you can't overwrite it.. meaning if any more changes are needed before this goes to production we have to skip 3.5.1 straight to 3.5.2. 

Also looks like this was pushed to npm, but not tagged in git?

**Comment by rjmackay** [comment]
@middle8media was this supposed to just fix #78 or more? 
You've got other changes of brandons in this branch somehow. 

Also can we please please have descriptive branch names. `f-revisions` is *not* descriptive. Everything is a revision.. what are you revising?

File: package.json (hunk 1 above)
Comment (middle8media) [inline package.json]: this was published before your documentation was written. I will reach out if needed.

**Comment by middle8media** [comment]
> @middle8media was this supposed to just fix #78 or more?

Just #78 

> You've got other changes of brandons in this branch somehow.
Sorry about that, Brandon and I need to review and get back in sync

> Also can we please please have descriptive branch names. f-revisions is not descriptive. Everything is a revision.. what are you revising?

This branch was created to include small changes as opposed to an actual feature. I thought `f-revisions` would suffice. I supposed I should have just called it simply `small-revisions` or `ui-fixes`

**Comment by rjmackay** [comment]
> This branch was created to include small changes as opposed to an actual feature. I thought f-revisions would suffice. I supposed I should have just called it simply small-revisions or ui-fixes

Not really. Neither of those is useful info.  Something like `mode-context-fixes` or `fix-mode-context-overflow` would both provide some info. Part of the issue with `ui-fixes` or `revisions` is that we've had 10+ branches named like that over time..

**Comment by middle8media** [comment]
> Not really. Neither of those is useful info. Something like mode-context-fixes or fix-mode-context-overflow would both provide some info. Part of the issue with ui-fixes or revisions is that we've had 10+ branches named like that over time..

Ok, I will ger uber specific next time. Thanks.

**Comment by rjmackay** [comment]
Cherrypick the relevant commit direct to master. Closing this PR

**Pull request closed without merging**